### PR TITLE
Implement exit codes test [full ci]

### LINKED ISSUE #1680: make sure we have a test for the exit codes
#1514

### PR DESCRIPTION
**Pull request by mhagen-vmware**
Fixes #1680

### CODE REVIEW

File: tests/test-cases/Group1-Docker-Commands/1-6-Docker-Run.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ This test requires that a vSphere server is running and available
 10. Issue docker run --link busy3:busy3 busybox ping -c2 busy3 to the VIC appliance
 11. Issue docker run -it busybox /bin/df to the VIC appliance
 12. Issue docker run -d -p 6379 redis:alpine to the VIC appliance
+13. Issue docker run -it busybox /bin/true
+14 Issue docker run -it busybox /bin/false
```
Comment (gigawhitlocks) [inline tests/test-cases/Group1-Docker-Commands/1-6-Docker-Run.md]: The missing period here isn't an issue is it?

**Comment by gigawhitlocks** [comment]
Probably wanna add in that missing period and then this LGTM

**Comment by matthewavery** [comment]
lgtm

**Comment by mhagen-vmware** [comment]
.

**Pull request merged**